### DISCUSSION TITLE
[FEATURE] Add /setwork hand ''price''

### DISCUSSION THREAD

**Discussion by SkyDude0** [Zrips/CMI#6899] [Enhancement]
I suggest to add a way to set custom item, like with different name,lore etc.. on the /worth list

With the command /setwork hand ''price'' or something like that

**Comment by Zrips** [comment]
Custom model data is being recognized currently, so you can use that as way to separate one item over the other and set different prices for it. Using items lore or name isint best solution as it can be modified by the user, especially name. 
You can modify items custom model data with **/cmi itemcmdata** or any other plugin made for this.

**Reply by SkyDude0** [reply]
i konw, but my player cant modify the lore, its more about this. I dont use custom model, just custom item like MMOitems or special head with custom lore

**Reply by mrfloris** [reply]
> i konw, but my player cant modify the lore, its more about this. I dont use custom model, just custom item like MMOitems or special head with custom lore

You can make a custom alias that runs the itemname and itemlore commands as console on the item they're holding.
